acceptance/api/v1/application_batch_delete_test.go

```diff
@@ -0,0 +1,106 @@
+package v1_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/epinio/epinio/acceptance/helpers/catalog"
+	api "github.com/epinio/epinio/internal/api/v1"
+	"github.com/epinio/epinio/pkg/api/core/v1/models"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("AppBatchDelete Endpoint", func() {
+	var (
+		namespace, app1, app2 string
+	)
+	containerImageURL := "splatform/sample-app"
+
+	BeforeEach(func() {
+		namespace = catalog.NewNamespaceName()
+		env.SetupAndTargetNamespace(namespace)
+
+		for _, a := range []*string{&app1, &app2} {
+			*a = catalog.NewAppName()
+			env.MakeContainerImageApp(*a, 1, containerImageURL)
+
+			configurationName := *a + "-conf"
+			env.MakeConfiguration(configurationName)
+			env.BindAppConfiguration(app1, configurationName, namespace)
+		}
```

Comment:
Why the pointers?

Reply:
Because I need to assign a value to the existing variables on line 30 (maybe this helps: https://go.dev/play/p/dm5q7SQO6_A)